app/controllers/api/v1/mentors_controller.rb

```diff
@@ -13,12 +15,17 @@ def index
 
   def show
     id = params[:id]
-    render json: MentorSerializer.new(Mentor.find_by_id(id))
+    render json: MentorSerializer.new(Mentor.find(id))
   end
 
   def update
     id = params[:id]
-    Mentor.find_by_id(id).update(mentor_params)
+    Mentor.find(id).update(mentor_params)
```

Comment:
after updating, mentor serializer will need to return the updated mentor

Reply:
Will add the following:     render json: MentorSerializer.new(Mentor.find(id))

Reply:
thanks!

Reply:
thanks!